xapian-applications/omega/templates/styles.css

```diff
@@ -0,0 +1,128 @@
+body {
+    margin: 0px;
+}
+
+fieldset {
+    outline: none;
+    border: 0px;
+    text-align: center;
+}
+
+fieldset.query-search, .query-metadata,
+.query-suggested-container, .query-options, .query-suggestion {
+    background-color: #FAFAFA;
+    text-align: center;
+    display: block;
+}
+
+.query-metadata {
+    margin: 0px;
+    box-shadow: 0 5px 5px -6px black;
+}
+
+input[type=search].query-search {
+    width: 400px;
+    height: 35px;
+    font-size: 14px;
+    padding: 20px;
+    margin-right: 5px;
+    border: 0px solid #aaa;
+    border-radius: 2px;
+    box-shadow: 0 2px 2px 0 rgba(0,0,0,0.16), 0 0 0 1px rgba(0,0,0,0.08);
+}
+
+input[type=search]:focus.query-search, input[type=search]:hover.query-search{
+    outline: none;
+    box-shadow: 0 3px 8px 0 rgba(0,0,0,0.2), 0 0 0 1px rgba(0,0,0,0.08);
+}
+
+input[type=submit].query-search {
+    height: 40px;
+    width: 100px;
+    color: #313131;
+    font-size: 14px;
+    border: 0px;
+    background-color: #ECEFF1;
+    border: 1px solid #ddd;
+}
+
+input[type=submit]:hover.query-search {
+    transition-duration: .4s;
+    background-color: #FAFAFA;
+    color: #111111;
+    border: 1px solid #bbb;
+}
+
+td.result-data, td.result-metadata {
+    padding: 20px;
+}
+
+td.result-metadata {
+    background-color: #ecf0f1;
+}
+
+div.result-metadata {
+    display: block;
+    font-size: 12px;
+}
+
+.result-data {
+    display: block;
+    font-size: 14px;
+}
+
+td.result-data {
+    box-shadow: 0 2px 2px 0 rgba(0,0,0,0.16), 0 0 0 1px rgba(0,0,0,0.08);
+}
+
+header.result-data {
+    font-size: 18px;
+}
+
+.result-metadata-progress-container {
+    border: 1px solid #888;
+    border-radius: 3px;
+    float: left;
+    width:60px;
+    height:10px;
+}
+
+.result-metadata-progress {
+    height: 10px;
+    background-color: #ff5252;
+}
+
+ul.pagination {
+    width: 100%;
+    display: inline-block;
+    margin: 4px;
+    padding: 0;
+    text-align: center;
+}
+
+ul.pagination li {
+    display: inline;
+    margin: 0;
+}
+
+ul.pagination li input {
+    background-color: white;
+    border: 0;
+    border-radius: 5px;
+    padding: 10px;
+    border: 1px solid #ddd;
+}
+
+ul.pagination li input:disabled {
+    background-color: #757575;
+    color: white;
+}
+
+ul.pagination li input:hover {
+    background-color: #546E7A;
+}
+
+footer {
+    float: right;
+    margin: 2px;
+}
```

Comment:
It's always a good idea (unless the file format forbids it) to have a newline at the end of the file, because git works more smoothly that way when people edit around there subsequently.